keyboards/handwired/ms_sculpt_mobile/handwired/rules.mk

```diff
@@ -0,0 +1,39 @@
+ifdef ASTAR
+	CFLAGS=-D ASTAR
+	MCU = atmega32u4
+	SCULPT_UPLOAD_COMMAND = avrdude -p $(MCU) -c avr109 -U flash:w:$(TARGET).hex -P $(USB)
```

Comment:
These commands aren't necessary; you should just be able to use the `:flash` make target and QMK will call the right flashing tool based on the `MCU` and `BOOTLOADER` defined.